src/middleware/compression.ts

```
@@ -0,0 +1,217 @@
+        return true;
+    }
+
+    /**
+     * Create Express middleware
+     */
+    public middleware() {
+        return (req: Request, res: Response, next: NextFunction): void => {
+            // Skip if client doesn't accept compression
+            const acceptEncoding = req.get('Accept-Encoding') || '';
+            if (!acceptEncoding.includes('gzip') && !acceptEncoding.includes('deflate')) {
+                return next();
```

Biome (2.1.2)

[error] The function should not return a value because its return type is void.

The function is here:

'void' signals the absence of value. The returned value is likely to be ignored by the caller.

(lint/correctness/noVoidTypeReturn)